app/_locales/en/messages.json

```
@@ -764,10 +764,10 @@
     "message": "Contract requesting signature"
   },
   "contractRequestingSpendingCap": {
-    "message": "Contract requesting spending cap"
+    "message": "Third party requesting spending cap"
   },
   "contractTitle": {
-    "message": "Contract details"
+    "message": "Third party details"
```

Add hyphen right @coreyjanssen ?
```suggestion
    "message": "Third-party details"
```

---

yep! add hyphen here please